src/main/java/website/frontrow/artificial/intelligence/PathCalculation.java

```diff
@@ -9,7 +9,6 @@
  * given the player information, a list of enemies and the current level.
  */
 @SuppressWarnings("checkstyle:hideutilityclassconstructor")
```

Comment:
Again if the checkstyle is fixed, remove the suprression, if it isn't keep the explanation.

<layout